apps/sms/style/sms.css

```diff
@@ -962,14 +962,20 @@ section[role="region"][class*='-information'] #messages-compose-form {
 }
 
 /* Phone details label styles */
-.phone-type,
+.phone-type {
+  display: -moz-inline-box;
```

Comment:
this is not a standard value for "display".

You can use "display: inline-block" with "vertical-align: bottom" and it works fine in my test.